modules/ducktests/tests/ignitetest/tests/control_utility/consistency_test.py

```diff
@@ -108,3 +113,83 @@ def test_logging(self, ignite_version):
             raise IgniteExecutionException("Fail.")
         except TimeoutError:
             pass
+
+    @cluster(num_nodes=4)
+    @ignite_versions(str(DEV_BRANCH))
+    @defaults(backups=[1], cache_count=[1], entry_count=[50_000], entry_size=[50_000], preloaders=[1])
+    def test_consistency_check_performance(self, ignite_version, backups, cache_count, entry_count, entry_size,
+                                           preloaders):
+        """
+        Tests time of performing consistency check and idle_verify utilities over the same data.
+        """
+
+        data_gen_params = DataGenerationParams(backups=backups, cache_count=cache_count, entry_count=entry_count,
+                                               entry_size=entry_size, preloaders=preloaders)
+
+        node_config = IgniteConfiguration(
+            version=IgniteVersion(ignite_version),
+            cluster_state="INACTIVE",
+            include_event_types=[EventType.EVT_CONSISTENCY_VIOLATION],
+            data_storage=DataStorageConfiguration(
+                max_wal_archive_size=2 * data_gen_params.data_region_max_size,
+                default=DataRegionConfiguration(
+                    persistence_enabled=True,
+                    max_size=data_gen_params.data_region_max_size
+                )
+            ),
```

Comment:
Do we really need 

- max_wal_archive_size
- max_size
- metric_exporters

here ?

Reply:
no